src/views/home/components/list/list.view.tsx

```diff
@@ -28,64 +27,45 @@ interface ListProps {
   onClose: () => void;
 }
 
-const filterChainList = (chainList: ChainList, value: string): ChainList => ({
-  ...chainList,
-  items: chainList.items.filter((chain) => chain.name.toUpperCase().includes(value.toUpperCase())),
-});
-
-const filterTokenList = (tokenList: TokenList, value: string): TokenList => ({
-  ...tokenList,
-  items: tokenList.items.filter(
-    (token) =>
-      token.name.toUpperCase().includes(value.toUpperCase()) ||
-      token.symbol.toUpperCase().includes(value.toUpperCase())
-  ),
-});
-
-const List: FC<ListProps> = ({ placeholder, list, onClose }) => {
+const List: FC<ListProps> = ({ list, onClose }) => {
   const classes = useListStyles();
-  const [values, setValues] = useState(list);
-
-  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const value = event.target.value.trim();
-    const filtered: List =
-      list.type === "chain" ? filterChainList(list, value) : filterTokenList(list, value);
-
-    setValues(filtered);
-  };
+  const [values] = useState(list);
+  const inputRef = useRef<HTMLInputElement>(null);
```

Comment:
it's not used after remove SearchBox

Reply:
I guess this was to put the focus back on the SearchBox. How's it that the SearchBox has been removed?

Reply:
Because you see all the supported chains without use SearchBox.

Reply:
Fair enough, but then why did we add a filter? 🤣

Reply:
Design and the box was reused in Tokenlist

Reply:
Removed!